api/cmd/managerapi.go

```diff
@@ -93,13 +102,16 @@ func NewManagerAPICommand() *cobra.Command {
 				return newMws
 			}
 
+			fmt.Printf("DEBUG-ETCDCONFIG: %+v\n\n", *conf.ETCDConfig)
```

Comment:
please remove debug code

Reply:
yes yes. Just 1 min.
![image](https://user-images.githubusercontent.com/41498427/114654459-d5aabb00-9d07-11eb-82f4-d5d64e28ea48.png)
Actually, it was the root cause. The endpoint was different, haha.